script/Arcueid/Day 5/05_00_ARC05_9_2.txt

```diff
@@ -122,7 +124,7 @@ After a brief moment of bewilderment,
 -- Page 650, Offset 10405.
 -- それでもいいのね？　もう逃げる事もできないわよ？」
 Is that okay with you?
- There's no escaping after that, you know?"
+ That means we don't have the option of escaping if it fails."
```

Comment:
Sounds tonally wrong
maybe change "escaping" to "running away" in the original, but other than that I don't see why you would change it.